modules/network-observability-total-resource-usage.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+// * network_observability/configuring_operator.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-total-resource-usage-table_{context}"]
+= Total average memory and CPU usage
+
+The following table outlines averages of total resource usage for clusters with a sampling value of 1, 50, and 400 for 3 different tests: `Test 1`, `Test 2`, and `Test 3`. The tests differ in the following ways:
+
+- `Test 1` takes into account the total number of namespace, pods and services in {product-title} cluster, places load on the eBPF agent, and represents use cases with a high number of workloads for a given cluster size. For example, `Test 1` consists of 76 Namespaces, 5153 Pods, and 2305 Services.
+- `Test 2` takes into account a high ingress traffic throughput.
+- `Test 3` takes into account the total number of namespace, pods and services in {product-title} cluster, places load on the eBPF agent on a larger scale than `Test 1`, and represents use cases with a high number of workloads for a given cluster size. For example, `Test 3` consists of 553 Namespaces, 6998 Pods, and 2508 Services.
+
+Since different types of cluster use cases are exemplified in the different tests, the numbers in this table cannot be linearly compared side-by-side, but instead are intended to be used as a benchmark for evaluating your personal cluster usage.
+
+[NOTE]
+====
+Metrics exported to Prometheus can impact the resource usage. Cardinality values for the metrics can help determine how much resources are impacted.. For more information, see Flows format reference.
+====
+
+[IMPORTANT]
+====
+The examples outlined in the table demonstrate scenarios that are tailored to specific workloads. Consider each example only as a baseline from which adjustments can be made to accommodate your workload needs.
+====
+
+.Total average resource usage
```

Comment:
Maybe reference each test in the table? In other words, instead of just providing sampling value as indicator of which test is which, instead identify test 1, 2 and 3?